mmv1/third_party/terraform/services/container/resource_container_node_pool.go.erb

```diff
@@ -191,6 +191,11 @@ var schemaNodePool = map[string]*schema.Schema{
 					Required:     true,
 					Description:  `Type defines the type of placement policy`,
 				},
+				"policy_name": {
```

Comment:
Can the policy be updated in-place? Can you add a step to the test that triggers that update?

If not, you can put ForceNew: true on this schema and it will force recreation if the field changes

Reply:
The policy can't be updated, it's create-only. Added the `ForceNew: true`, thanks!